pkg/config/config_test.go

```diff
@@ -256,7 +256,7 @@ var _ = Describe("Config", func() {
 			defaultConfig, err := readConfigFromFile("testdata/containers_default.conf", conf)
 
 			OCIRuntimeMap := map[string][]string{
-				"runc": []string{
```

Comment:
I didn't realize this was superflous to go....

Reply:
that was an automated fix from my vim-go setup. it really helps to keep an eye on lints and fmt